tests/golos.referral_tests.cpp

```diff
@@ -159,7 +162,10 @@ BOOST_FIXTURE_TEST_CASE(close_referral_tests, golos_referral_tester) try {
     init_params();
 
     auto expire = 8; // sec
-    BOOST_CHECK_EQUAL(success(), referral.create_referral(N(issuer), N(sania), 500, cur_time().to_seconds() + expire, asset(50000, symbol(3, "GLS"))));
+
+    update_cur_time();
```

Comment:
It is not clear why we should do `update_cur_time` in this place. It's better to write tests in such a way that they are extremely clear and do not contain any subtle points.
I would suggest getting rid of calling this function, and implement `cur_time` function without caching time value.